### PR TITLE
Fixes support for transparent/skipover pixels

### PR DESCRIPTION
**Pull request by RandomGamer123**
Makes pixels with color (69, 42, 0) actually skipped again and adds it to README, just a mild annoyance really since this can easily be changed locally, but still something good to keep working.

### CODE REVIEW

**Comment by Ferotiq** [comment]
They removed it again?

**Comment by RandomGamer123** [comment]
> They removed it again?

Yeah accidentally

Also updated PR since I was on old version until now which still used RGB colors and not RGBA

**Comment by Ferotiq** [comment]
I wonder why we aren't using just standard png transparent pixels instead.

File: main.py [hunk 1]
```diff
@@ -397,11 +397,11 @@ def get_unset_pixel(self, x, y, index):
                     "{}, {}, {}, {}",
                     pix2[x + self.pixel_x_start, y + self.pixel_y_start],
                     new_rgb,
-                    new_rgb != (69, 42, 0),
```
Comment (thoroc) [inline main.py:400]: Since that is used in at least 2 places, you could make a constant

**Comment by thoroc** [comment]
> I wonder why we aren't using just standard png transparent pixels instead.

I am not familiar with pillow, and who ever done the work, probably hasn't realised either. Feel free to submit a new PR to remediate this short coming. :)

**Review by thoroc — APPROVED** [review]

**Comment by Ferotiq** [comment]
can this get merged soon?

**Comment by maltejur** [comment]
Now images with transparent pixels don't work anymore https://github.com/rdeepak2002/reddit-place-script-2022/issues/307

**Pull request merged**